Tests/RSADecryptionTests.swift

```diff
@@ -122,7 +124,7 @@ class RSADecryptionTests: RSACryptoTestCase {
             return
         }
 
-        for algorithm in KeyManagementAlgorithm.allCases {
+        for algorithm in keyManagementModeAlgorithms {
             guard algorithm != .direct else {
                 continue
             }
```

Comment:
Can't you omit all these `guard not direct` checks now that `keyManagementModeAlgorithms` has fixed contents?

Reply:
Good catch. Removed.